init.rb

```diff
@@ -86,6 +87,7 @@ class << Setting
 SearchController.helper(FullTextSearch::Hooks::SearchHelper)
 SearchController.prepend(FullTextSearch::Hooks::ControllerSearchIndex)
 IssuesController.helper(FullTextSearch::Hooks::SimilarIssuesHelper)
+IssueQuery.prepend(FullTextSearch::IssueAnySearchable)
```

Comment:
We use `FullTextSearch::Hooks` for modules that overwrite Redmine classes.

Reply:
fix: [246de36](https://github.com/clear-code/redmine_full_text_search/pull/164/commits/246de36d45a245ebd8204548ba78c15ee6d4514f) I moved it under `hooks/`